src/pytorch_lightning/_graveyard/__init__.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import pytorch_lightning._graveyard.callbacks
+import pytorch_lightning._graveyard.cloud_io
```

Comment:
Why did you choose `cloud_io`? This has nothing todo with our `src/lightning_lite/utilities/cloud_io.py` module, no?

I would suggest
```suggestion
import pytorch_lightning._graveyard.legacy_import_unpickler
```

Or something along those lines

(remember the test name too)

Reply:
Fine with me as well. I chose `cloud_io` since our basic load function is within the `cloud_io` package.